volume/store-utils.go

```diff
@@ -56,13 +56,13 @@ func DeleteVolume(name string) error {
 
 //GetVolumes retrives the json objects from the store and converts them into
 //respective volinfo objects
-func GetVolumes() ([]Volinfo, error) {
+func GetVolumes() ([]Volume_list, error) {
```

Comment:
Almost all go projects follow lower camel cases. Could you change the variable name to something like volList?